internal/file/file.go

```diff
@@ -42,14 +48,16 @@ func GetFilename(path string) string {
 	return filename
 }
 
-func ReadHCLFile(path string) *hclwrite.File {
+func ReadHCLFile(path string) (*hclwrite.File, error) {
 	src, err := ioutil.ReadFile(path)
-	errors.PanicOnError(err, nil)
+	if err != nil {
+		return nil, err
+	}
 
 	file, diagnostics := hclwrite.ParseConfig(src, path, hcl.InitialPos)
-	if diagnostics.HasErrors() {
-		hclErrors := diagnostics.Errs()
-		log.Fatalln(hclErrors)
+	if err := multierr.Combine(diagnostics.Errs()...); err != nil {
```

Comment:
cleaner way to combine multiple errors.
If there are no errors (list is empty) err will be nil (Therefore there is no need to call HasErrors()).